package.json

```diff
@@ -26,9 +26,9 @@
   },
   "dependencies": {
     "@justeat/f-logger": "0.7.2",
-    "@justeat/f-utils": "1.0.0-beta.0",
-    "@justeat/fozzie": "4.0.0-beta.0",
-    "@justeat/fozzie-colour-palette": "3.0.0-beta.6",
+    "@justeat/f-utils": "2.0.0",
+    "@justeat/fozzie": "5.0.0-beta.10",
+    "@justeat/pie-design-tokens": "0.19.0",
```

Comment:
There shouldn't be a need to add `@justeat/pie-design-tokens` and "@justeat/f-utils" separately because they should be available through the fozzie package, if I am not mistaken

Reply:
I was a bit scared to remove them tbh! (as they were there already) I'll try removing though, as like you say, they shouldn't be needed…

Reply:
Have updated this now 👍